Testsuite update:

YCP ignores the integer parameter in y2log() functions
(https://github.com/yast/yast-core/blob/master/libycp/src/YCPBuiltinMisc.cc#L558)
while in Ruby it is handled properly
(https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/ycp/logger.rb#L8)

This can break tests which use Report
(https://github.com/yast/yast-yast2/blob/master/library/modules/Report.ycp#L544)

In this case fixing the reporting means that some errors (originally) from
Report surface even though testedfiles has only Snapper

diff --git a/agent-snapper/conf/Makefile.am b/agent-snapper/conf/Makefile.am
index 40c39fb..e69de29 100644
--- a/agent-snapper/conf/Makefile.am
+++ b/agent-snapper/conf/Makefile.am
@@ -1,7 +0,0 @@
-#
-# Makefile.am for .../agent-snapper/conf
-#
-
-scrconf_DATA = snapper.scr
-
-EXTRA_DIST = $(scrconf_DATA)
diff --git a/agent-snapper/doc/autodocs/Makefile.am b/agent-snapper/doc/autodocs/Makefile.am
index 3ca9c78..a599bae 100644
--- a/agent-snapper/doc/autodocs/Makefile.am
+++ b/agent-snapper/doc/autodocs/Makefile.am
@@ -2,5 +2,7 @@
 # Makefile.am for .../agent-snapper/doc/autodocs
 #
 
-AUTODOCS_SUBDIR=agent-snapper
-include $(top_srcdir)/autodocs-cc.ami
+# TODO FIXME: temporarily disabled
+
+#AUTODOCS_SUBDIR=agent-snapper
+#include $(top_srcdir)/autodocs-cc.ami
diff --git a/doc/autodocs/Makefile.am b/doc/autodocs/Makefile.am
index 2cbc75e..17633f0 100644
--- a/doc/autodocs/Makefile.am
+++ b/doc/autodocs/Makefile.am
@@ -1,3 +1,5 @@
 # Makefile.am for snapper/doc/autodocs
 
-include $(top_srcdir)/autodocs-ycp.ami
+# TODO FIXME: temporarily disabled
+
+#include $(top_srcdir)/autodocs-ycp.ami
diff --git a/testsuite/Makefile.am b/testsuite/Makefile.am
index d9b12b0..e85fbc8 100644
--- a/testsuite/Makefile.am
+++ b/testsuite/Makefile.am
@@ -5,7 +5,7 @@
 #
 
 AUTOMAKE_OPTIONS = dejagnu
-EXTRA_DIST = $(wildcard tests/*.out) $(wildcard tests/*.err) $(wildcard tests/*.ycp) $(wildcard tests/*.yh)
+EXTRA_DIST = $(wildcard tests/*.out) $(wildcard tests/*.err) $(wildcard tests/*.rb)
 
 testsuite_prepare = @ydatadir@/testsuite/Makefile.testsuite
 
diff --git a/testsuite/tests/GetSnapshotPath.out b/testsuite/tests/GetSnapshotPath.out
index 0fbf6ff..b1893ce 100644
--- a/testsuite/tests/GetSnapshotPath.out
+++ b/testsuite/tests/GetSnapshotPath.out
@@ -1,4 +1,5 @@
 Read	.snapper.path $["num":0] nil
+Log	Snapshot '0' was not found.
 Return	
 Read	.snapper.path $["num":0] "/snapshots/0/snapshot"
 Return	/snapshots/0/snapshot
diff --git a/testsuite/tests/ReadConfigs.out b/testsuite/tests/ReadConfigs.out
index 3809ebc..4d03d8d 100644
--- a/testsuite/tests/ReadConfigs.out
+++ b/testsuite/tests/ReadConfigs.out
@@ -2,6 +2,7 @@ Read	.snapper.configs []
 Return	[]
 Return	root
 Read	.snapper.configs nil
+Log	File /etc/sysconfig/snapper is not available.
 Return	["root"]
 Return	root
 Read	.snapper.configs ["opt", "var", "root"]
diff --git a/yast2-snapper.spec.in b/yast2-snapper.spec.in
index c91664f..62db558 100644
--- a/yast2-snapper.spec.in
+++ b/yast2-snapper.spec.in
@@ -8,6 +8,8 @@ BuildRequires:	libsnapper-devel >= 0.0.11
 BuildRequires:	yast2-core-devel >= 2.23.1
 BuildRequires:	libtool doxygen gcc-c++ perl-XML-Writer
 
+Requires:       yast2-ruby-bindings >= 1.0.0
+
 Summary:	YaST - file system snapshots review
 
 %description
@@ -25,7 +27,7 @@ YaST module for accessing and managing btrfs system snapshots
 %defattr(-,root,root)
 %dir @yncludedir@/snapper
 @yncludedir@/snapper/*
-@clientdir@/snapper.ycp
+@clientdir@/snapper.rb
 @moduledir@/Snapper.*
 @desktopdir@/snapper.desktop
 @scrconfdir@/*.scr
